modules/vcf2maf/1.1/vcf2maf.smk

```diff
@@ -92,17 +92,23 @@ rule _vcf2maf_run:
         > {log.stdout} 2> {log.stderr}
         """)
 
+
+def get_chain(wildcards):
+    if "38" in str({wildcards.genome_build}):
+        return reference_files("genomes/{genome_build}/chains/hg38ToHg19.over.chain")
+    else:
+        return reference_files("genomes/{genome_build}/chains/hg19ToHg38.over.chain")
+
 rule _vcf2maf_crossmap:
     input:
         maf = rules._vcf2maf_run.output.maf,
-        convert_coord = CFG["inputs"]["convert_coord"]
+        convert_coord = CFG["inputs"]["convert_coord"],
+        chains = get_chain
     output:
         maf =  CFG["dirs"]["crossmap"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{base_name}.converted.maf"
```

Comment:
I am worried that .converted.maf is not very clear and, if anything, the path name implies the wrong genome build for the converted maf. I get that it's implied that the .converted.maf is the opposite genome build as the directory implies. Should we discuss a separate directory that states the new/converted build? Maybe we can discuss this in the GAMBL meeting

Reply:
interested in your thoughts, Kostia and @lkhilton